components/ai_chat/core/browser/conversation_handler.cc

```diff
@@ -969,6 +969,13 @@ void ConversationHandler::UpdateOrCreateLastAssistantEntry(
     entry->text = event->get_completion_event()->completion;
   }
 
+  if (event->is_selected_language_event()) {
+    OnSelectedLanguageChanged(
+        event->get_selected_language_event()->selected_language);
+    // Don't add this event to history
```

Comment:
We miss a test case for this by checking `GetConversationHistory()`

Reply:
Added in 6726cfb701f5db5b9bea897c071d8a41f068c24f